packages/block-editor/src/components/block-list/block.js

```diff
@@ -87,7 +87,7 @@ function BlockListBlock( {
 		isDraggingBlocks && ( isSelected || isPartOfMultiSelection );
 
 	// Determine whether the block has props to apply to the wrapper.
-	if ( ! lightBlockWrapper && blockType.getEditWrapperProps ) {
```

Comment:
I'm not understanding why we are removing the "! lightBlockWrapper" check.

Reply:
the intent before this PR was that `getEditWrapperProps` should be deprecated and not used with the new light wrapper (used by group). That's why the check is removed.

As I said here, ideally we'd have a better API to extend the block wrapper props https://github.com/WordPress/gutenberg/pull/21021#discussion_r395111645